composer.json

```diff
@@ -19,13 +19,13 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0|^8.1",
-        "xeroapi/xero-php-oauth2": "^2.0|^3.0|^4.0|^5.0|^6.0|^7.0",
-        "laravel/framework": "^6.0|^7.0|^8.0|^9.0|^10.0|^11.0"
+        "php": "^8.2|^8.3",
+        "xeroapi/xero-php-oauth2": "^7.0",
```

Comment:
Happy to revert this to ^2.0, but thought it'd be good to get everyone on the latest version while we're forcing framework changes etc.

Reply:
Yeah, I think it's reasonable.
I've been considering whether we should start marking major xeroapi/xero-php-oauth2 releases as a major release for this package too. Thoughts?

Reply:
I guess in theory we should, I have a restraint in my own composer.json for `xeroapi/xero-php-oauth2` but I suppose in theory, users could just do a `composer update` and get the new major version of the package.